File: CS50X/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>
void wall(int M)
{
    for (int Z = 0; Z < M; Z++)
    {
        printf("#");
    }
}
void spaces(int M1)
{
    for (int Z1 = 0; Z1 < M1; Z1++)
    {
        printf(" ");
    }
}
int main(void)
{
    int ask;
    do
    {
        ask = get_int("Height: ");
    } while (ask < 1 || ask > 8);

    for (int n = 1; n < ask + 1; n++)
    {
        spaces(ask - n);
        wall(n);
        printf("  ");
        wall(n);
        printf("\n");
    }
}

File: CS50X/sentimental-mario-more/mario.py
# TODO
import cs50 as cs

while True:
    input = cs.get_int("Height: ")
    if input >= 1 and input <= 8:
        break

for i in range(input):
    z = input
    while z > i+1:
        print(" ", end="")
        z -= 1
    for i in range(z):
        print("#", end="")
    print("  ", end="")
    for i in range(z):
        print("#", end="")
    print("")
